lightning/src/ln/channelmanager.rs

```diff
@@ -424,6 +424,18 @@ pub(super) enum RAACommitmentOrder {
 	RevokeAndACKFirst,
 }
 
+/// Information about a payment which is currently being claimed.
+struct PendingClaimingPayment {
```

Comment:
By that logic, I think we should rename the `pending_claimed_htlcs` map as well. Although I'm a bit confused because it seems like we have sent the `update_fulfill`, which to me means we've "claimed"